CovertInfiltration/Src/CovertInfiltration/Classes/X2InfiltrationModTemplate.uc

```diff
@@ -11,23 +11,22 @@ class X2InfiltrationModTemplate extends X2DataTemplate;
 
 var int HoursAdded; // to the covert infiltration time
 var int Deterrence; // how much this item reduces the risk of injury/capture on non-Infil CAs
-var name MultCategory; // other equipped items with a ItemCat matching this have their HoursAdded multiplied by this item's InfilMultiplier
-var float InfilMultiplier;
+var EInfilModifierType ModifyType; // what kind of game element does this template modify
+var name ElementName; // the name of the game element this template modifies
 
 function bool ValidateTemplate(out string strError)
 {
-	if (MultCategory == '' && InfilMultiplier != 1)
+	switch (ModifyType)
 	{
-		strError = "CI: Error in templates, InfilMultiplier has been set but MultiplierCategory is blank.";
-		return false;
+		case eIMT_Item:
+		case eIMT_Category:
+		case eIMT_Ability:
+		case eIMT_Character:
+			return true;
+		default:
+			strError = "CI: Error in templates, an X2InfiltrationModTemplate does not have a ModifyType which should not be possible!";

```

Comment:
`strError` will automatically be prefixed with "[template class] [template data name]:", so here you would need to write only:
```suggestion
			strError = "invalid ModifyType";
```